src/htm/algorithms/SpatialPooler.cpp

```diff
@@ -176,18 +176,6 @@ void SpatialPooler::setBoostStrength(Real boostStrength) {
   boostStrength_ = boostStrength;
 }
 
-UInt SpatialPooler::getIterationNum() const { return iterationNum_; }
-
-void SpatialPooler::setIterationNum(UInt iterationNum) {
```

Comment:
SP removed `iteration` stuff. not really needed. And connections will have the iteration from #537